types/index.test-d.ts

```diff
@@ -21,6 +21,7 @@ const expectResponse = function (res: Response) {
   expectAssignable<Function>(res.json)
   expectAssignable<http.ServerResponse>(res.raw.res)
   expectAssignable<http.IncomingMessage>(res.raw.req)
+  expectAssignable<Record<string, string>>(Object.fromEntries(res.cookies.map(x => [x.name, x.value])))
```

Comment:
very complicated...  and only checks name and value

why not 

expectType<Date | undefined>(res.cookies[0].expires